runtime/compiler/p/codegen/PPCPrivateLinkage.cpp

```diff
@@ -2738,29 +2737,69 @@ void buildCRC32CCall(TR::Node *callNode,
       deps->getPostConditions()->setDependencyInfo(map.getTargetIndex(TR::RealRegister::gr7), wasteArg, TR::RealRegister::gr7, UsesDependentRegister);
       }
 
-   TR::Register *gr0Reg = deps->searchPreConditionRegister(TR::RealRegister::gr0);
-   TR::Register *gr11Reg = deps->searchPreConditionRegister(TR::RealRegister::gr11);
-   TR::Register *gr12Reg = deps->searchPreConditionRegister(TR::RealRegister::gr12);
+   if (comp->compileRelocatableCode())
+      {
+
+      TR::SymbolReference * vpmsumSymRef = (comp)->getSymRefTab()->findOrCreateRuntimeHelper(TR_PPCcrc32_vpmsum,false,false,true);
```

Comment:
This should be `findOrCreateRuntimeHelper(TR_PPCcrc32_vpmsum,false,false,false);` since this helper will not preserve all registers.